src/organisms/gv-newsletter-subscription-page.js

```diff
@@ -0,0 +1,276 @@
+              @gv-input:input="${this._onInputChange}"
+              type="email"
+              required
+              .value="${this.email}"
+              .disabled="${this.emailDisabled}"
+              placeholder="${this.emailPlaceholder}"
+            ></gv-input>
+            <gv-button @click="${this._onSubscribe}"
+                       id="email-submit"
+                       provider="graviteeio_am"
+                       icon-right="navigation:angle-right"
+                       .disabled=${!this.emailDisabled}>
+              ${this.subscribeLabel}
+            </gv-button>
+          </div>
+          ${this.taglines.length > 0 ? html`
+          <div class="newsletter__taglines">
+            ${this.taglines.map((item, i, arr) => html`
+              <div class="newsletter__tagline ${i === arr.length - 1 ? 'last' : ''}">
+                <div class="newsletter__tagline-text">
+                  ${item}
+                </div>
+              </div>
+            `)}
+          </div>`
+          : ''}
+          <div class="newsletter__skip">
+            <gv-button @click="${this._onSkip}"
+                       provider="graviteeio_am"
+                       icon-right="navigation:angle-right"
+                       small
+                       outlined>
+              ${this.skipLabel}
+            </gv-button>
+          </div>
+          <div class="social">
+            <gv-button class="social-link"
+                       link
+                       icon="thirdparty:github"
+                       .href="${this.githubUrl}"
+                       @gv-button:click="${this._navigateSocial.bind(this, this.githubUrl)}">
+            </gv-button>
+            <gv-button class="social-link"
+                       link icon="thirdparty:twitter"
+                       .href="${this.twitterUrl}"
+                       @gv-button:click="${this._navigateSocial.bind(this, this.twitterUrl)}">
+            </gv-button>
+            <gv-button class="social-link"
+                       link
+                       icon="thirdparty:linkedin"
+                       .href="${this.linkedinUrl}"
+                       @gv-button:click="${this._navigateSocial.bind(this, this.linkedinUrl)}">
+            </gv-button>
+            </gv-link>
+          </div>
+        </div>
+    `;
+  }
+
+  _onSubscribe () {
+    dispatchCustomEvent(this, 'subscribe', this.email);
+  }
+
+  _onSkip () {
+    dispatchCustomEvent(this, 'skip');
+  }
+
+  _navigateSocial (url) {
+    window.open(url, '_blank');
+  }
+
+  _onInputChange ({ detail }) {
+    setTimeout(() => {
+      this.email = detail;
+      const emailInput = this.shadowRoot.querySelector('#email');
+      const emailSubmit = this.shadowRoot.querySelector('gv-button#email-submit');
+      if (emailInput.invalid && !this.emailDisabled) {
+        emailSubmit.setAttribute('disabled', true);
+      }
+      else {
+        emailSubmit.removeAttribute('disabled');
+      }
+    });
```

Comment:
You can use lit-element lifecycle and use a property for disable the submit button